code/game/machinery/computer/camera_advanced.dm

```diff
@@ -70,6 +71,12 @@
 	current_user = null
 	return ..()
 
+/obj/machinery/computer/camera_advanced/add_context(atom/source, list/context, obj/item/held_item, mob/living/user)
+	if(isnull(held_item))
+		context[SCREENTIP_CONTEXT_LMB] = "Use console"
+		return CONTEXTUAL_SCREENTIP_SET
+	return NONE
```

Comment:
This seems like unnecessary context unless we want to add it to all computers

Reply:
oh true I suppose

Reply:
yeah leaving all that for a separate pr with computer disassembly context